src/cli/cli.cpp

```diff
@@ -5700,6 +5774,35 @@ template <> otError Interpreter::Process<Cmd("panid")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli parent
+ * @code
+ * parent
+ * Ext Addr: be1857c6c21dce55
+ * Rloc: 5c00
+ * Link Quality In: 3
+ * Link Quality Out: 3
+ * Age: 20
+ * Version: 4
+ * Done
+ * @endcode
+ * @code
+ * CSL clock accuracy: 20
+ * CSL uncertainty: 5
+ * @endcode
+ * @par api_copy
+ * #
+ * @par
+ * Get the diagnostic information for a Thread Router as parent.
+ * @par
+ * Note: When operating as a Thread Router when OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE is enabled, this command
+ * will return the cached information from when the device was previously attached as a Thread Child. Returning cached
+ * information is necessary to support the Thread Test Harness - Test Scenario 8.2.x requests the former parent
+ * (i.e. Joiner Router's) MAC address even if the device has already promoted to a router.
+ * @par
+ * Note: When OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE is enabled, this command will return two extra lines with
```

Comment:
Remove the Note: from this, just leave it as a normal paragraph, it doesn't look good for all the text to be a note on the website